### PR TITLE
feat: Add experiments array to job resource

### PR DESCRIPTION
**Pull request by kerenfi**
| Question                | Answer                                                                          |
| ---------------- | -------------------------------------------------------------------------- |
| Bug fix         | ✖                                                                        |
| New feature     | ✔                                                                |
| Breaking change | ✖                                                                        |
| Deprecations    | ✖                                                                        |
| Documentation   | ✖                                                                        |
| Tests added     | ✔                                                                      |

Related issues: #634

### CODE REVIEW

File: src/chaos-experiments/models/chaosExperimentsManager.js [hunk 1]
```diff
@@ -72,3 +76,10 @@ module.exports.updateChaosExperiment = async function (experimentId, chaosExperi
     await databaseConnector.updateChaosExperiment(experimentId, chaosExperiment);
     return chaosExperiment;
 };
+
+module.exports.getChaosExperimentsByIds = async function (experimentIds, exclude) {
```
Comment (GuyAb) [inline src/chaos-experiments/models/chaosExperimentsManager.js:80]: why do you have this function twice with same name here?

File: src/jobs/helpers/jobVerifier.js [hunk 2]
```diff
@@ -68,3 +75,27 @@ module.exports.verifyTestExists = async (req, res, next) => {
...
+    const jobPlatform = await configHandler.getConfigValue(CONFIG.JOB_PLATFORM);
+    const experiments = jobBody.experiments;
+    if (!experiments || experiments.length === 0) {
+        next();
+    } else if (experiments.length > 0 && jobPlatform.toUpperCase() !== KUBERNETES){
+        errorToThrow = new Error(ERROR_MESSAGES.CHAOS_EXPERIMENT_SUPPORTED_ONLY_IN_KUBERNETES);
+        errorToThrow.statusCode = 400;
+        next(errorToThrow);
+    } else {
+        const experimentIds = new Set(experiments.map(experiment => experiment.experiment_id));
```
Comment (GuyAb) [inline src/jobs/helpers/jobVerifier.js:91]: would suggest do them both (91 92) as one liner, more clear

File: tests/integration-tests/jobs/createJobKubernetes-test.js [hunk 3]
```diff
@@ -136,6 +138,92 @@ describe('Create job specific kubernetes tests', async function () {
...
+                    it('Get the jobs, without one_time query param, only cron job should be returned', async () => {
+                        getJobsFromService = await schedulerRequestCreator.getJobs({
+                            'Content-Type': 'application/json'
+                        });
+
+                        should(getJobsFromService.status).eql(200);
+
+                        const relevantJobs = getJobsFromService = getJobsFromService.body.filter(job => job.id === cronJobId || job.id === oneTimeJobId);
+                        should(relevantJobs.length).eql(1);
+                        should(relevantJobs[0].id).eql(cronJobId);
```
Comment (GuyAb) [inline tests/integration-tests/jobs/createJobKubernetes-test.js:218]: what exactly do you intend to check in this test? That the API still works well? Or do these tests should be extended when additional logics enters?

File: src/jobs/helpers/jobVerifier.js [hunk 4]
```diff
@@ -11,7 +17,8 @@ const consts = require('../../common/consts');
  */
 function verifyCronExpression(exp) {
     try {
-        const ct = new CronTime(exp);
+        // eslint-disable-next-line no-unused-vars
+        const _ct = new CronTime(exp);
```
[the remark this answers is not in the document]
Comment (kerenfi) [reply src/jobs/helpers/jobVerifier.js]: it was an eslint comment, and it is not advised to do new Object without assigning

**Pull request merged**